contracts/strategies/JoeStrategyV2.sol

```diff
@@ -6,6 +6,7 @@ import "../interfaces/IJoeChef.sol";
 import "../interfaces/IPair.sol";
 import "../interfaces/IWAVAX.sol";
 import "../interfaces/IERC20.sol";
+import "../lib/DexInteractionHandler.sol";
```

Comment:
Instead of applying those changes to JoeStrategyV2, please create a new JoeStrategyV3